### PR TITLE
[Xedra Evolved] fix spells, that give temp limbs

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
None
#### Purpose of change
Accidentally found i overengineered this spells
#### Describe the solution
Remove the mutation step, and give limb directly from effect
#### Testing
Now it works

### CODE REVIEW

File: data/mods/Xedra_Evolved/effects/effects.json [hunk 1]
```diff
@@ -1033,15 +1033,13 @@
...
-    "points": 1,
-    "visibility": 10,
-    "description": "Ethereal hands appear behind you back, making you looks like an ancient god.",
-    "valid": false,
-    "starting_trait": false,
-    "purifiable": false,
+    "type": "effect_type",
+    "id": "spell_karma_arms_eff",
+    "name": [ "Karma" ],
+    "desc": [ "Ethereal hands appear behind you back, making you looks like an ancient god" ],
```
Comment (b3brodie) [inline data/mods/Xedra_Evolved/effects/effects.json:1039]: you -> your .  End with a period.

File: data/mods/Xedra_Evolved/effects/effects.json [hunk 2]
```diff
@@ -1033,15 +1033,13 @@
...
+    "apply_message": "Ethereal, transparent arms grow from your shoulders, and <color_cyan>glow</color>.",
```
Comment (b3brodie) [inline data/mods/Xedra_Evolved/effects/effects.json:1040]: Remove comma

File: data/mods/Xedra_Evolved/effects/effects.json [hunk 3]
```diff
@@ -1033,15 +1033,13 @@
...
+    "remove_message": "The cyan arms grow dim, and finally disappear.",
```
Comment (b3brodie) [inline data/mods/Xedra_Evolved/effects/effects.json:1041]: Remove comma or add "they" between and and finally

File: data/mods/Xedra_Evolved/effects/effects.json [hunk 4]
```diff
@@ -1054,61 +1052,31 @@
...
-    "enchantments": [ { "condition": "ALWAYS", "modified_bodyparts": [ { "gain": "stalker_eyes" } ] } ]
-  },
   {
     "type": "effect_type",
     "id": "stalker_eyes_eff",
-    "name": [ "" ],
-    "desc": [ "" ],
+    "name": [ "Stalker's Vision" ],
+    "desc": [
+      "Your own eyes transform into the orbits of an unknown creature.  You see much better, both in the light and in the dark"
```
Comment (b3brodie) [inline data/mods/Xedra_Evolved/effects/effects.json:1060]: Remove "own" as it duplicates meaning from the starting "Your".  For the second sentence, "You now see much better in the light and dark." flows better

File: data/mods/Xedra_Evolved/effects/effects.json [hunk 5]
```diff
@@ -1054,61 +1052,31 @@
...
+    "enchantments": [ { "condition": "ALWAYS", "modified_bodyparts": [ { "gain": "stalker_eyes" } ] } ]
   },
   {
     "type": "effect_type",
     "id": "devil_tail_eff",
-    "name": [ "" ],
-    "desc": [ "" ],
+    "name": [ "Devil's Tail" ],
+    "desc": [
+      "Not actually devil's, this long tail with triangle point in the end can stretch for a lot, and could easily kill someone nearby"
```
Comment (b3brodie) [inline data/mods/Xedra_Evolved/effects/effects.json:1074]: "While not actually a devil's, this long, triangular pointed tail can stretch very far, allowing it to easily kill someone nearby."

File: data/mods/Xedra_Evolved/effects/effects.json (hunk 1 above)
Comment (GuardianDll) [reply data/mods/Xedra_Evolved/effects/effects.json:1039]: to make a suggestion, you can press this small button, and edit the text inside
image

```suggestion
    "desc": [ "Ethereal hands appear behind your back, making you looks like an ancient god." ],
```
image

**Pull request merged**